modules/publicize/publicize.php

```diff
@@ -97,18 +97,15 @@ function get_profile_link( $service_name, $c ) {
 		$cmeta = $this->get_connection_meta( $c );
 
 		if ( isset( $cmeta['connection_data']['meta']['link'] ) ) {
-			if ( 'facebook' == $service_name && 0 === strpos( parse_url( $cmeta['connection_data']['meta']['link'], PHP_URL_PATH ), '/app_scoped_user_id/' ) ) {
```

Comment:
Thanks - missed this one, was meaning to strip it out along with the other Facebook changes and ask @aduth what is/isn't safe to have in our build script anymore.